app/helpers/form_errors_builder.rb

```diff
@@ -0,0 +1,29 @@
+class FormErrorsBuilder < ActionView::Helpers::FormBuilder
```

Comment:
What do we gain from this implementation at the cost of complexity over a simple shared errors partial such as this generic one https://github.com/NikitaAvvakumov/blog/blob/master/app/views/shared/_error_messages.html.erb?